.travis.yml

```diff
@@ -203,7 +203,19 @@ script:
         make check-integration || exit 1
         make check-themes || exit 1
         do_codecov functionaltests
-        do_codecov_gcov c_code
+
+        # Run gcov.  Upload for codecov directly, gets done later for coveralls.
+        # NOTE: files for the main target have the build dir as prefix, but
+        #       otherwise it is the source directory?!
```

Comment:
Any idea about this?
Is it relevant where the binary is run from?
Probably rather where it is built from?!